### PR TITLE
post-processing xarray attributes updates

### PR DESCRIPTION
**Pull request by ryancoe**
## Description
I noticed while working on a case-study and making the plots that we should fix the following:

- [x] set the `attrs = {'long_name': ...}` whenever possible (e.g., for `time` and `omega`)
- [x] units for `ds.omega` should be `rad/s` not `(rad)`
- [x] Rerun the examples so they show these changes in their plots

## Type of PR
Minor update

## Checklist for PR
- [x] Authors read the [contribution guidelines](https://github.com/SNL-WaterPower/WecOptTool/blob/main/.github/CONTRIBUTING.md)
- [x] The pull request is from an issue branch (not main) on your fork to the [main branch in WecOptTool](https://github.com/SNL-WaterPower/WecOptTool). 
- [x] The authors have given the admins edit access
- [x] All changes adhere to the style guide including PEP8, Docstrings, and Type Hints.
- ~~Added note in [Changelog](https://github.com/SNL-WaterPower/WecOptTool/blob/main/CHANGES.md)~~
- ~~Modified the documentation if applicable~~
- ~~Modified or added a new test~~
- [x] All tests pass
- [x] Documentation builds
- ~~[Reference or close any relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)~~

### CODE REVIEW

File: wecopttool/core.py [hunk 1]
```diff
@@ -838,7 +838,8 @@ def _post_process_wec_dynamics(self,
         # xarray - time domain
         dims_td = ('time', 'influenced_dof')
         coords_td = [
-            (dims_td[0], self.time, {'units': 's'}),
+            (dims_td[0], self.time, {'units': 's',
+                                     'long_name':'Time'}),
             (dims_td[1], self.hydro.influenced_dof.values)]
         attrs_pos = {'long_name': 'WEC position', 'units': 'm or (rad)'}
         attrs_vel = {'long_name': 'WEC velocity', 'units': 'm/s or (rad)/s'}
```
Comment (cmichelenstrofer) [inline wecopttool/core.py:845]: (rad) needs to be changed here too? maybe missed it in other places too (do "find/replace all"?)

File: wecopttool/pto.py [hunk 2]
```diff
@@ -193,10 +193,11 @@ def post_process(self, wec: WEC, x_wec: npt.ArrayLike, x_opt: npt.ArrayLike
...
         # assemble time-domain xarray
         dims = ['time', 'dof_pto']
-        coords = [(dims[0], wec.time, {'units': 's'}), (dims[1], self.names)]
+        coords = [(dims[0], wec.time, {'units': 's',
+                                       'long_name':'Time'}), (dims[1], self.names)]
         attrs_f = {'long_name': 'PTO force', 'units': 'N or N*m'}
         attrs_p = {'long_name': 'PTO power', 'units': 'W'}
-        attrs_pos = {'long_name': 'PTO position', 'units': 'm or (rad)'}
+        attrs_pos = {'long_name': 'PTO position', 'units': 'm or rad'}
         attrs_vel = {'long_name': 'PTO velocity', 'units': 'm/s or (rad)/s'}
```
Comment (cmichelenstrofer) [inline wecopttool/pto.py:201]: this (rad) should be replaced too. Just do "find all"

**Review by cmichelenstrofer — CHANGES_REQUESTED** [review]
good changes but you missed a few "(rad)".

File: wecopttool/core.py (hunk 1 above)
Comment (ryancoe) [reply wecopttool/core.py]: e3317ba0a22dad9b77b8568d4cb571bf4ce02555

File: wecopttool/pto.py (hunk 2 above)
Comment (ryancoe) [reply wecopttool/pto.py]: e3317ba0a22dad9b77b8568d4cb571bf4ce02555

**Review by cmichelenstrofer — APPROVED** [review]

**Pull request merged**